data/global.lua

```diff
@@ -46,6 +46,13 @@ table.contains = function(array, value)
 	return false
 end
 
+table.merge = function(t1,t2)
```

Comment:
`function(t1,t2)`
you should make a separation after the comma
example:
`function(t1, t2)`